controllers/datadogagent/component/clusteragent/utils.go

```diff
@@ -69,3 +72,37 @@ func GetClusterAgentService(dda metav1.Object) *corev1.Service {
 
 	return service
 }
+
+// GetMetricsServerServiceName returns the external metrics provider service name
+func GetMetricsServerServiceName(dda metav1.Object) string {
+	return fmt.Sprintf("%s-%s", dda.GetName(), apicommon.DefaultMetricsServerResourceSuffix)
+}
+
+// GetMetricsServerAPIServiceName returns the external metrics provider apiservice name
+func GetMetricsServerAPIServiceName() string {
+	return "v1beta1.external.metrics.k8s.io"
```

Comment:
i know it was like this before your PR; i'm curious why we don't use a constant

Reply:
I wasn't sure about that either so I left it like that before, but I'll change it to a constant